src/WSRequestHandler_SceneCollections.cpp

```diff
@@ -2,6 +2,11 @@
 
 #include "WSRequestHandler.h"
 
+/**
+* @typedef {Object} `ScenesCollection`
```

Comment:
Now you may want to use this in the spec of `ListSceneCollections`

Reply:
Sounds like a good idea... :)
(sorry again for this messy thing, I am not used to github)